taxi/tests/test_forms.py

```diff
@@ -0,0 +1,141 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.forms import DriverCreationForm
+from taxi.models import Manufacturer, Car, Driver
+
+
+class FormTests(TestCase):
+    def test_driver_creation_form_with_license_number_first_last_name_is_valid(self):
+        form_data = {
+            "username": "New_Driver",
+            "password1": "TestPassword123",
+            "password2": "TestPassword123",
+            "first_name": "TestFirstName",
+            "last_name": "TestLastName",
+            "license_number": "AAA55555"
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
```

Comment:
The assertion `self.assertEqual(form.cleaned_data, form_data)` may not work as expected because `form.cleaned_data` will not include the password fields and might have additional fields like `id`. Consider checking only the fields that are expected to be in `cleaned_data`.